### PR TITLE
[GSOC] Question Answering

### PR DESCRIPTION
**Pull request by fajemila**
It demonstrates how to convert and use the [DistillBERT](https://huggingface.co/distilbert-base-cased-distilled-squad) model for question Answering with OpenVINO.

### CODE REVIEW

**Comment by fajemila** [comment]
hi @adrianboguszewski , this is the new PR

**Comment by adrianboguszewski** [comment]
I'm copying my review from #912 

Please:
- update your branch
- move your notebook to 213-question-answering directory
- rename the 213-question-answering notebook to 213-bert-question-answering
- avoid using HTML, replace it with markdown
- use OpenVINO model inside answer_question function. Now, you're using the Pytorch model
- use Grammarly or any other tool to fix grammar, typos and punctuations

**Comment by adrianboguszewski** [comment]
Ok, I can see now that you're passing the infer_request. As it's not easy to spot, please use the openvino_model inside the function without passing infer_request. The infer_request is not necessary. You can call the model directly. See 002-openvino-api notebook for the reference.

**Comment by fajemila** [comment]
> Ok, I can see now that you're passing the infer_request. As it's not easy to spot, please use the openvino_model inside the function without passing infer_request. The infer_request is not necessary. You can call the model directly. See 002-openvino-api notebook for the reference.

Okay

**Comment by fajemila** [comment]
> * update your branch

Hi, I was trying to update my branch the last time by synchronizing my fork with the latest branch, which made me close my PR unknowingly. Now,  trying to update my branch by rebasing my commits on top of it,  I will have to be resolving some conflicts that occurred during the rebase. Is this the best approach?

**Comment by adrianboguszewski** [comment]
There are 2 approaches:
1. Rebase your branch and place your commits on the top - it would require you to push with force
2. Merge the main branch into your branch, resolve conflicts and push - recommended

**Comment by fajemila** [comment]
> There are 2 approaches:
> 
> 1. Rebase your branch and place your commits on the top - it would require you to push with force
> 2. Merge the main branch into your branch, resolve conflicts and push - recommended

yess,  i just got this to work, i will go ahead to do finish up.

File: notebooks/213-question-answering/213-bert-question-answering.ipynb [hunk 1]
```diff
@@ -0,0 +1,440 @@
+{
```
Comment (OpenVINO-dev-contest) [inline notebooks/213-question-answering/213-bert-question-answering.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #13.        result = model.infer(inputs=inputs)
</pre><p>Do you mean <code></code><code>complied_model</code><code></code> here ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1022/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (fajemila) [reply notebooks/213-question-answering/213-bert-question-answering.ipynb:1]: Yes, I have just pushed the correct one

**Comment by adrianboguszewski** [comment]
@fajemila, please check all files you're committing. I can see you change much more than your notebook. Please fix this first. Your contribution should contain just your notebook and anything listed in the self-check list:
- [ ] each function is described by docstrings and type hints
- [ ] notebook contains explicit descriptions and explanatory diagrams
- [ ] the notebook doesn't use any data (image, video, etc.) that is not CC4.0 licensed 
- [ ] there is a README.md file in consistent style (look at other notebooks)
- [ ] the notebook is added to the main README
- [ ] there are no grammar, punctuation or typo issues (use any free tool for that e.g. Grammarly)
- [ ] there are no committed files besides notebook and readme (please use images or videos from data dir)
- [ ] your PR doesn't change any other notebooks
- [ ] all CI checks passed

**Comment by adrianboguszewski** [comment]
I'm closing this due to inactivity for a long time. Nevertheless, thank you for contributing and applying for Google Summer of Code. We encourage you to consider GSoC 2024 as well :)

**Pull request closed without merging**